src/MainWindow.vala

```diff
@@ -464,6 +467,7 @@ namespace Terminal {
             menu_popover_grid.add (color_grid);
             menu_popover_grid.add (new Gtk.Separator (Gtk.Orientation.HORIZONTAL));
             menu_popover_grid.add (natural_copy_paste_button);
+            menu_popover_grid.add (terminate_process_button);
```

Comment:
The settings menu probably isn't the right place for this. All of the other menu items here effect all of your tabs.

A more appropriate location is probably either the tab menu or the context menu